src/components/Home/Buttons.jsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
 import FlatButton from 'material-ui/FlatButton';
+import { connect } from 'react-redux';
 import strings from 'lang';
 import { IconSteam } from 'components/Icons';
 import { ButtonsDiv } from './Styled';
 
-export default () => (
+const mapStateToProps = (state) => {
```

Comment:
I like to put mapstatetoprops at the bottom (next to the connect) but that's just style